packages/chains/src/linea.ts

```diff
@@ -0,0 +1,35 @@
+import { Chain } from './types'
+
+export const linea = {
+  id: 59_144,
+  name: 'Linea Mainnet',
+  network: 'linea-mainnet',
+  nativeCurrency: { name: 'Linea Ether', symbol: 'ETH', decimals: 18 },
+  rpcUrls: {
+    infura: {
+      http: ['https://linea-mainnet.infura.io/v3'],
+      webSocket: ['wss://linea-mainnet.infura.io/ws/v3'],
+    },
+    default: {
+      http: ['https://rpc.linea.build'],
+      webSocket: ['wss://rpc.linea.build'],
+    },
+    public: {
+      http: ['https://rpc.linea.build'],
+      webSocket: ['wss://rpc.linea.build'],
+    },
+  },
+  blockExplorers: {
+    default: {
+      name: 'Etherscan',
+      url: 'https://lineascan.build',
+    },
+  },
+  contracts: {
+    multicall3: {
+      address: '0xca11bde05977b3631167028862be2a173976ca11',
```

Comment:
All multicall contracts get deployed to this address, but that's something that must be verified.